java/tools/slice-tools/README.md

```diff
@@ -0,0 +1,193 @@
+# Ice Slice Tools Gradle Plugin
+
+The **Ice Slice Tools Gradle Plugin** integrates the **Slice-to-Java compiler (`slice2java`)** into Gradle-based
```

Comment:
I would call this code the "Slice Tools for Ice and Java". No need for Gradle in the name. 

Likewise, the other integrations would be the "Slice Tools for Ice and C#" and the "Slice Tools for Ice and C++".

Reply:
I think including Gradle / MSBuild etc. in the plugin/integration README title makes the name hard to read / understand.

In practice, we only provide a single integration per language, so there is not much value in including it in the title:
- C# => MSBuild
- C++ => MSBuild
- Java => Gradle

I prefer "Slice Tools for Ice" over "Ice Slice Tools" as the README title. "Slice Tools for Ice Gradle Plugin" has too many words.

My concern with "Slice Tools for Ice" as a README title is I don't want to give the impression that there is a single "Slice Tools for Ice" code/plugin/integration that works for Java, C# and C++. We provide programming language - specific integrations.

We can fix these README titles in a follow-up PR.

Reply:
This is not really true. We have CMake support as well. We also have the builders for maven and ant. We may want to update these to work with Ice 3.8.